### PR TITLE
feat(DIA-114): add highlights and achievements to artist about tab

### PR DESCRIPTION
**Pull request by araujobarret**
This PR resolves [DIA-114] 

### Description

Adds Highlights and Achievements section. Some logic was copied from Force to keep 1:1 behavior consistent among the 2 platforms.

iPhone 14 Pro Max

https://github.com/artsy/eigen/assets/15792853/f31ecd8e-b35e-4f4f-9a1f-57d23438182d

iPad

Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-09-05 at 15 47 03

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- add highlights and achievements to artist about tab - araujobarret

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[DIA-114]: https://artsyproduct.atlassian.net/browse/DIA-114?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Components/Artist/ArtistAbout/ArtistCareerHighlights.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  }
+`
+
+interface ExpandableProps {
+  label?: string
+  expanded?: boolean
+  children: React.ReactNode
+}
+
+const Expandable: React.FC<ExpandableProps> = ({ children, expanded: propExpanded, label }) => {
```
Comment (araujobarret) [inline src/app/Components/Artist/ArtistAbout/ArtistCareerHighlights.tsx:61]: I don't see the need to put it in the palette, maybe if we bump into the need we can add it later

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (add highlights and achievements to artist about tab - araujobarret)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 39916e97390e192366ad906dffd67790774acc42
</p>

File: src/app/Components/Artist/Biography.tsx [hunk 2]
```diff
@@ -23,7 +25,7 @@ export const Biography: React.FC<BiographyProps> = ({ artist }) => {
   const canExpand = text.length > MAX_CHARS
 
   return (
-    <Flex maxWidth={650}>
+    <Flex maxWidth={MAX_WIDTH}>
```
Comment (araujobarret) [inline src/app/Components/Artist/Biography.tsx:28]: will meet with Ryan this week to check the best alternative for tablets here(bio and highlights), it's not optimal right now

File: src/app/Components/Artist/ArtistAbout/ArtistCareerHighlights.tsx (hunk 1 above)
Comment (gkartalis) [reply src/app/Components/Artist/ArtistAbout/ArtistCareerHighlights.tsx:61]: should we move this to another file though?

**Review by gkartalis — DISMISSED** [review]

File: src/app/Components/Artist/ArtistAbout/ArtistCareerHighlights.tsx (hunk 1 above)
Comment (araujobarret) [reply src/app/Components/Artist/ArtistAbout/ArtistCareerHighlights.tsx:61]: will move and add some comment for people if they want to use it and it becomes more like a shared element to move to palette 👍

Comment (araujobarret) [reply src/app/Components/Artist/ArtistAbout/ArtistCareerHighlights.tsx]: done in https://github.com/artsy/eigen/pull/9218/commits/768f310396fd0ff379af36a8b9dd37d32f9d0bb8

**Pull request merged**